### DISCUSSION TITLE
graphql/codegen/plugins/typescript/typed-document-node

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#10156] [Docs Discussions]
# graphql/codegen/plugins/typescript/typed-document-node

GraphQL-Codegen Documentation

https://the-guild.dev/graphql/codegen/plugins/typescript/typed-document-node

**Comment by kyle-villeneuve** [comment]
Is it possible to generate a hash for each generated document for use with persisted queries?
